htdocs/xoops_lib/Xoops/Core/Logger.php

```diff
@@ -204,7 +204,7 @@ private function reportFatalError($msg)
         if ('cli' === php_sapi_name()) {
             fprintf(STDERR, "\nError : %s\n", $msg);
         } else {
-            printf(_XOOPS_FATAL_MESSAGE, XOOPS_URL, $msg);
+            printf(_XOOPS_FATAL_MESSAGE, XOOPS_URL, utf8_encode($msg));
```

Comment:
`utf8_encode()` only works for ISO-8859-1.

Are you sure your files are UTF-8 encoded? That is our standard now; it is part of [PSR-2](http://www.php-fig.org/psr/psr-2/) which we try to follow, especially with new code.